src/lsqecc/simulation/qiskit_opflow_utils.py

```diff
@@ -15,11 +15,11 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
 # USA
 
-from typing import Dict, List, Optional
+import qiskit.quantum_info as qkinfo
 
 import qiskit.aqua.operators as qk
-import qiskit.exceptions as qkexcept
-import qiskit.quantum_info as qkinfo
+from qiskit.quantum_info import *
```

Comment:
Looks like `pre-commit.ci` si not clever enough to figure this change. Fair enough, though. The occurrences of `qkinfo` in the code are lost - I'd say it's beyond the scope of an import linter to catch it.

Maybe worth testing again without this `*` change?

Reply:
I don't think `isort` will fix `import *` issues, but from the [run report](https://github.com/latticesurgery-com/lattice-surgery-compiler/pull/144) - can be access by clicking Details, `flake8` catches it which failed the checks.

Reply:
oh if you are talking about `qkinfo`, isort just moved it to a different position.

Reply:
... I see, are you going to try it again with a different issue?

Reply:
I meant to say that `pre-commit.ci autofix` dosen't seem to fix many things, even after running it, the CI has errors

Reply:
NVM I get it, I thought black and isort were supposed to revert the change you introduced.

Reply:
Yep as I mentioned in https://github.com/latticesurgery-com/lattice-surgery-compiler/pull/147#issuecomment-966790546, `pre-commit.ci autofix` will only push changes made by black and isort only which does not guarantee that errors flagged by other hooks (mainly flake8 and mypy) will be fixed.